src/MainWindow.ui

```diff
@@ -211,25 +211,25 @@ You can drag SQL statements from an object row and drop them into other applicat
              <widget class="QComboBox" name="comboboxPragmaAutoVacuum">
               <item>
                <property name="text">
-                <string notr="true">None</string>
+                <string>None</string>
```

Comment:
Please restore the `notr`attribute for this and the other valuesso that they are never changed in tralations, otherwise the pragma edition will be broken.

If you still want to translate the values, you'd need to change also the code to make it work even with translated values.

Reply:
I understand that changing the values in the combo-boxes could break the code. Thanks for having explained.
I have restored the notr="true" attribute for them.
I hope the modification is now acceptable and that you can see it.